### DISCUSSION TITLE
Can pull the addon whit a big change?

### DISCUSSION THREAD

**Discussion by KeilAwk** [LuaLS/LLS-Addons#133] [Q&A] [answered]
I updated the commuity addon in the [repo](https://github.com/ravenfieldcommunity/ravenscriptidea), i didnt pull it here
I afraid that addon manager can't allow lots of  folder structure change, like a other addon before...
Help me check it plz, thx!

**Comment by carsakiller** [comment]
I'm not positive what you mean. You have two addons in the repo it seems? One at `RavenM/` and the other at `Main/`? The addon manager uses Git, and so can handle large changes in addons just fine... I am just not sure that I understand how your repo is laid out as each addon needs their own repo and it seems you have multiple addons in one repo.

**Comment by KeilAwk** [comment, marked as answer]
> I'm not positive what you mean. You have two addons in the repo it seems? One at `RavenM/` and the other at `Main/`? The addon manager uses Git, and so can handle large changes in addons just fine... I am just not sure that I understand how your repo is laid out as each addon needs their own repo and it seems you have multiple addons in one repo.

Hmm...
Before the two commits, the repo only had one add-on. I moved the previous addon to `Main/` and put the added addon in `RavenM/`.
So the addon manager can handle it well, right?

**Reply by carsakiller** [reply]
Each addon needs its own separate repository. You can put the new addon in a different repo so that both can be available in the addon manager, or you can remove the old addon from the repo and from LLS-Addons.

**Comment by KeilAwk** [comment]
can't put two addon in one repo?
ok i get it
